spec/lib/os_places_api/client_spec.rb

```diff
@@ -182,6 +182,62 @@
       end
     end
 
+    context "the postcode is outdated in the database" do
+      before do
+        Postcode.create(
+          postcode: postcode,
+          results: [
+            {
+              "DPA" => {
```

Comment:
It would be great to not need to duplicate all this JSON - if the JSON ever changes, we'd have to update it in multiple places. I'd suggest taking an approach like this:

```ruby
  arbitrary_value = -0.1
  old_results = os_places_api_results.dup
  old_results.first["DPA"]["LNG"] = arbitrary_value
  old_results.first["DPA"]["LAT"] = arbitrary_value
  Postcode.create(postcode: postcode, results: old_results)
```